PokeAlarm/Twitter/TwitterAlarm.py

```diff
@@ -70,8 +73,38 @@ def set_alert(self, settings, default):
 
     # Post Pokemon Status
     def send_alert(self, alert, info):
-        args = {"status": replace(alert['status'], info)}
-        try_sending(log, self.connect, "Twitter", self.__client.statuses.update, args)
+        # Optimize remaining tweet length when removing <gmaps>
+        # There must be spaces around <gmaps> for Twitter to recognize link properly
+        maps_link_included = True
+        max_length = MAX_TWEET_LEN - TWEET_URL_LEN
+        if ' <gmaps> ' in alert['status']:
```

Comment:
I'm not a big fan of this part. I think it would be better to just check for <gmaps> is located at the end of the string, then save it and carry it across.